distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.1");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv12() throws Exception {
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+        sslContext.init(null, null, null);
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        checkAuthenticationRequest(config, url);
+    }
+
+    @Test
+    public void testTLSv1() throws Exception {
+        String httpsProtocols = System.getProperties()
+                .getProperty("https.protocols");
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        try {
+            System.setProperty("https.protocols", "TLSv1");
+
+            SSLContext sslContext = SSLContext.getInstance("TLSv1");
+            sslContext.init(null, null, null);
+            SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext);
+
+            RestAssuredConfig config = new RestAssuredConfig();
+            config.sslConfig(SSLConfig.sslConfig()
+                    .sslSocketFactory(sslSocketFactory));
+
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+
+            given().config(config)
+                    .auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url)
+                    .then()
+                    .assertThat()
+                    .statusCode(401);
+        } finally {
+            System.setProperty("https.protocols", httpsProtocols);
+            configureRestForGuest(SDK_SOAP_CONTEXT);
+            getSecurityPolicy().waitForGuestAuthReady(url);
+        }
+    }
+
+    @Test
+    public void testAllowedCipherSuites() throws Exception {
+        List<String> supportedCipherSuites = Arrays.asList("TLS_DHE_RSA_WITH_AES_128_GCM_SHA256",
+                "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256",
+                "TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
+                "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+                "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256");
+
+        String originalCiphers = System.getProperty("https.cipherSuites");
+
+        try {
+            System.setProperty("https.cipherSuites", StringUtils.join(supportedCipherSuites, ","));
+
+            String[] supportedHttpProtocols = {"https"};
+
+            SSLContext sslContext = SSLContext.getDefault();
```

Comment:
Not necessary but it looks like you might be able to extract a helper method to create the SSLContext and RestAssuredConfig.